src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from django.utils.translation import gettext_lazy as _
+
+from openforms.plugins.registry import BaseRegistry
+from openforms.variables.base import BaseStaticVariable
+from openforms.variables.constants import FormVariableDataTypes
+
+if TYPE_CHECKING:
+    from openforms.submissions.models import Submission
+
+
+class Registry(BaseRegistry[BaseStaticVariable]):
+    """
+    A registry for the Objects API registration variables.
+    """
+
+    module = "objects_api"
+
+
+register = Registry()
+"""The Objects API registration variables registry."""
+
+
+@register("pdf_url")
+class PdfUrl(BaseStaticVariable):
+    name = _("PDF Url")
+    data_type = FormVariableDataTypes.string
+
+    def get_initial_value(self, submission: Submission | None = None):
+        return (
+            submission.objects_api_registration_data.pdf_url
+            if submission is not None
+            else None
+        )
```

Comment:
I much prefer guard clauses:

```suggestion
    def get_initial_value(self, submission: Submission | None = None):
        if submission is None:
             return None
        return submission.objects_api_registration_data.pdf_url
```
it makes it a lot clearer that the submission may be missing.